noxfile.py

```diff
@@ -347,17 +348,21 @@ def prerelease(session):
     session.run("python", "-m", "pip", "freeze")
 
     # Run all tests, except a few samples tests which require extra dependencies.
+
```

Comment:
nit: remove the added empty line?

Reply:
In this case, if it passes linting and black... I am fine with it being as it is.

Reply:
I'm not sure about this - if there isn't a good reason to change it, I don't think we should.